.travis.yml

```
@@ -0,0 +1,22 @@
+dist: xenial
+language: python
+python:
+  - "3.6"
+  - "3.7"
+  - "pypy3"
+os:
+  - linux
+before_script:
+  - python -m pip install pip --upgrade
+  - python -m pip list
+  - export PYTHONHASHSEED=${PYTHONHASHSEED:-${RANDOM}}
+  - echo "export PYTHONHASHSEED=${PYTHONHASHSEED}"
+  - python -m pip install .[test]
+  - python -m pip install codecov
+  - export COVERAGE_PROCESS_START=$(pwd)/.coveragerc
+  - export PYTEST_ADDOPTS=-v
+script:
+  - python -m flake8
+  - python $DEBUG -m coverage run pytest
```

After some testing, it turns out we want this:
```suggestion
  - python -m coverage run -m pytest
```